src/components/Cluster/ClusterDetail/Ingress/InstallIngressButton.tsx

```diff
@@ -158,15 +116,15 @@ const InstallIngressButton: React.FC<IInstallIngressButtonProps> = ({
 
       {!isLoading && (
         <Text>
-          {ingressApp ? (
+          {installedIngressApp ? (
             '🎉 Ingress controller installed. Please continue to the next step.'
           ) : (
             <>
               This will install the{' '}
               <StyledLink to={ingressAppDetailPath} href={ingressAppDetailPath}>
-                NGINX Ingress Controller app {ingressAppToInstall.version}
+                NGINX Ingress Controller app {ingressAppToInstall?.version}
```

Comment:
Are there things that could go wrong? 

How does the component handle failures to fetch or get the version?